scripts/bash/lxd-client

```diff
@@ -131,10 +131,12 @@ _have lxc && {
       lvm.vg_name rsync.bwlimit volatile.initial_source \
       volatile.pool.pristine volume.block.filesystem \
       volume.block.mount_options volume.size volume.zfs.remove_snapshots \
-      volume.zfs.use_refquota zfs.clone_copy zfs.pool_name"
```

Comment:
This should be in a separate commit, something along the lines of `scripts/bash: Add lvm.stripes keys`